scripts/exregional_run_analysis.sh

```diff
@@ -328,17 +337,20 @@ case $MACHINE in
 "WCOSS_C" | "WCOSS" | "WCOSS_DELL_P3")
    obsfileprefix=${obs_source}
    obspath_tmp=${OBSPATH}/${obs_source}.${YYYYMMDD}
-
   ;;
 "JET" | "HERA")
    obsfileprefix=${YYYYMMDDHH}.${obs_source}
    obspath_tmp=${OBSPATH}
-
+  ;;
+"ORION" )
+   obs_source=rap
+   #obsfileprefix=${YYYYMMDDHH}.${obs_source}
+   obsfileprefix=rap.${YYYYMMDD}/${obs_source}
+   obspath_tmp=${OBSPATH}
```

Comment:
Please remove commented code.

Also, please use `$obs_source` instead of `rap` in the `obsfileprefix` definition.

Reply:
1) That is not comment. That is one kind of data may be used on Orion. I added explanations on this.
2) Replaced "rap" with "$obs_source".